finality-aleph/src/party/mod.rs

```diff
@@ -432,9 +432,17 @@ where
     }
 
     pub async fn run(mut self) {
-        let last_finalized_number = self.client.info().finalized_number;
+        let mut finalized_number = self.client.info().finalized_number;
```

Comment:
Suggestion: move this into a seperate method -  it would keep that `run` method minimalistic and tidy + its name would be part of `documentation`.

Reply:
lgtm. Is there any reason for making it public?